libnopegl/src/pipeline.h

```diff
@@ -52,7 +52,6 @@ struct pipeline_uniform_desc {
 struct pipeline_texture_desc {
     char name[MAX_ID_LEN];
     int type;
-    int location;
```

Comment:
So only the location of the attributes remain in the pipeline descriptors; are those still necessary?

Reply:
It is. The terminology used for attribute is location not binding.